server/middleware/productauth.js

```diff
@@ -2,8 +2,8 @@ import express from 'express';
 
 import products from '../models/productsModel';
 
-export default class Productauth{
-    static async addproduct(req, res, next) {
+class Productauth{
```

Comment:
Why did you remove static keyword from the methods?

Reply:
removed it because i was exporting the class object  but seems unnecesary so i've reverted it back